scripts/build/build.js

```diff
@@ -29,7 +29,7 @@ function fitTerminal(input) {
   const columns = Math.min(process.stdout.columns || 40, 80);
   const WIDTH = columns - stringWidth(OK) + 1;
   if (input.length < WIDTH) {
-    input += Array(WIDTH - input.length).join(chalk.dim("."));
+    input += new Array(WIDTH - input.length).join(chalk.dim("."));
```

Comment:
We could also change stuff like this into `.repeat` on the `next` branch.

Reply:
/cc @fisker if you’re looking for more refactoring to do :)

Reply:
maybe later